src/color/Color.ts

```diff
@@ -279,7 +279,7 @@ export class Color {
    * @return {TRGBAColorSource | undefined} source
    */
   static sourceFromRgb(color: string): TRGBAColorSource | undefined {
-    const match = color.match(reRGBa);
+    const match = color.match(new RegExp(reRGBa, 'i'));
```

Comment:
the truth is that the best code is the literal regex inside the function here.
The constants outside the file are an organization thing that we pay with having those in the scope of the library, always initialized.